README.md

```diff
@@ -21,6 +21,7 @@
 
 ### Native
 
+0. Linux users: You may need to manually start PostgreSQL. Most major distributions use systemd as their init so all you need to do is run `systemctl start postgresql`
```

Comment:
What if we add a pre-amble to this section that points people to https://gorails.com/setup to get them 90% of the way there? There may be some discrepencies, but it seems like an easy way to help people get started.